.github/workflows/easy-cp.yml

```diff
@@ -60,7 +60,7 @@ jobs:
           base: ${{ env.RELEASE_BRANCH }}
           branch: cp-${{ env.CHANNEL }}-${{ env.COMMIT_SHA }}
           path: flutter
-          token: ${{ secrets.FLUTTERMIRRORINGBOT_TOKEN}}
```

Comment:
Let's use `secrets.FLUTTERACTIONSBOT_CP_TOKEN`

Reply:
Sure, thanks for adding the token!